metals/src/main/scala/scala/meta/internal/metals/ScalaTarget.scala

```diff
@@ -18,6 +20,17 @@ case class ScalaTarget(
     autoImports: Option[Seq[String]]
 ) {
 
+  def dialect: Dialect = {
+    scalaBinaryVersion match {
+      case _ if info.getDataKind() == "sbt" => Sbt
+      case "2.11" => Scala211
+      case "2.12" => Scala212
+      case "2.13" => Scala213
+      case version if version.startsWith("3.") => Scala3
```

Comment:
Do we also need to check for `startsWith("0")` still?

Reply:
So actually the parser will not work with those since the syntax changed. I think we should drop `0.` in this PR actually. Let's do a release with `2.12.13`, merge this and that should give enought time for EPFL students to finish their semester and never look at 0.27 version ever gain :sweat_smile: